policyengine_us/variables/household/expense/childcare/childcare_expenses_for_children_under_four.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class childcare_expenses_for_children_under_four(Variable):
```

Comment:
Sorry I didn't think of this earlier but 4 should actually be a policy parameter. So could you move this into `nyc/tax/income/credits/cdcc/nyc_age_restricted_childcare_expenses.py` after adding that?